### PR TITLE
Expand current syntax to support aliases for latest version (current/latest/node)

### PR DESCRIPTION
**Pull request by panticmilos**
**Description:**
This PR contains changes needed to support installing the latest node version from the distribution with multiple aliases. It also contains unit, e2e tests, and expanded documentation.

**Related issue:**
#418, #471

**Check list:**
- [ x ] Mark if documentation changes are required.
- [ x ] Mark if tests were added or updated to cover the changes.

### CODE REVIEW

**Comment by vsafonkin** [comment]
@panticmilos, please update PR title to be more readable :)

File: .github/workflows/versions.yml [hunk 1]
```diff
@@ -139,3 +139,33 @@ jobs:
...
+      - name: Retrieve version after install
+        run: |
+          updatedVersion=$(echo $(node --version))
+          echo "::set-output name=NODE_VERSION_UPDATED::$updatedVersion"
+        id: updatedVersion
+        shell: bash
+      - name: Compare versions
+        if: ${{ steps.version.outputs.NODE_VERSION == steps.updatedVersion.outputs.NODE_VERSION_UPDATED}}
+        run: |
+          echo "System node version ${{steps.updatedVersion.outputs.NODE_VERSION_UPDATED}} is same after the installation!"
```
Comment (vsafonkin) [inline .github/workflows/versions.yml:171]: Seems this test will be green for any case because we just write log to standard output or skip it.

Comment (panticmilos) [reply .github/workflows/versions.yml:171]: So as we discussed, we will change our approach in this e2e test.

**Review by vsafonkin — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

File: src/installer.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import * as tc from '@actions/tool-cache';
 import * as path from 'path';
 import * as semver from 'semver';
 import fs = require('fs');
+import * as installer from './installer';
```
[the remark this answers is not in the document]
Comment (Mohsen-Phoenix) [reply src/installer.ts]: approved these

**Pull request merged**